src/components/widgets/sales-opportunities-timeline/sales-opportunities-timeline.tmpl.html

```diff
@@ -0,0 +1,95 @@
+<div widget-sales-opportunities-timeline>
+
+  <div class="edit" ng-show="widget.isEditMode">
+    <h4>Widget settings</h4>
+    <div setting-organizations parent-widget="widget" class="part" deferred="::orgDeferred" />
+    <div setting-params-picker options="statusOptions" has-reach="hasReach" param="sales_stage_selection" parent-widget="widget" class="part" deferred="::paramsPickerDeferred1" />
+    <div setting-params-picker options="assigneesOptions" has-reach="hasReach" param="assignees_selection" parent-widget="widget" class="part" deferred="::paramsPickerDeferred2" />
```

Comment:
@an-pavlovych please remove this widget setting (assignees_selection), it is not needed.